playItAgain/css/app.css

```diff
@@ -0,0 +1,88 @@
+body{
+  background-color:orange;
+  font-family: "Monospace";
+}
+
+div{
+  padding: 20px;
+}
+
+#oscillator-list { /* Descendent Selector */
```

Comment:
I never use ids. They are not more useful than classnames and they cause weird side affects you might not be aware of. just change these to classnames and you're good. unique classnames!!